conanfile.py

```diff
@@ -38,6 +38,7 @@ def requirements(self):
         self.requires("boost/[~=1.69]", override=True)
         self.requires("fmt/[~=8.1.1]")
         self.requires("inja/[~=3.3.0]")
+        self.requires("nlohmann_json/[~=3.10.5]")
```

Comment:
Why don't we use `override=True` here since `nlohmann_json` is not a direct dependency?